internal/llrp/reader_functional_test.go

```diff
@@ -170,6 +170,7 @@ func writeCapture(dir string, idx uint32, result []byte, typ MessageType, decode
 	bfn := filepath.Join(dir, baseName+".bytes")
 	jfn := filepath.Join(dir, baseName+".json")
 
+	//nolint: gosec
```

Comment:
What does linter not like here?

Reply:
`internal/llrp/reader_functional_test.go:173:12: G306: Expect WriteFile permissions to be 0600 or less (gosec)
        if err := ioutil.WriteFile(bfn, result, 0644); err != nil {
                  ^
internal/llrp/reader_functional_test.go:186:12: G306: Expect WriteFile permissions to be 0600 or less (gosec)
        if err := ioutil.WriteFile(jfn, j, 0644); err != nil {
                  ^`

Reply:
Add `G306: Expect WriteFile permissions to be 0600 or less` to the `//nolint`

Reply:
I have added that comment to the `//nolint`